app/routes/claim/view-claim.js

```diff
@@ -452,19 +456,17 @@ function handleError (error, req, res, updateConflict, next) {
   }
 }
 
-function populateNewData (data, req) {
+function populateNewData (data, req, keepUnassignedChanges) {
```

Comment:
Maybe default to `false` to make it clearer, i.e. `keepUnassignedChanges = false`